app/src/main/kotlin/com/hippo/ehviewer/ui/main/DownloadCard.kt

```diff
@@ -173,12 +178,13 @@ fun DownloadCard(
                         )
                     }
                 }
+                val actionButtonSize = MaterialTheme.typography.labelLarge.lineHeight.value.dp * 2 + 2.dp
                 Row(
                     verticalAlignment = Alignment.CenterVertically,
                     modifier = Modifier.constrainAs(actionsRef) {
                         end.linkTo(parent.end)
                         bottom.linkTo(parent.bottom)
-                    },
+                    }.then(Modifier.size(actionButtonSize).offset(x = 8.dp))
```

Comment:
What's the explicit `then` for?

Reply:
This was needed in a previous version to bypass the 48dp minimum size requirement for IconButton. No longer needed now.